build.gradle

```diff
@@ -14,32 +14,21 @@
  * limitations under the License.
  */
 
-buildscript {
-  repositories {
-    jcenter()
-    maven { url "http://spinnaker.bintray.com/gradle" }
-    maven { url "https://plugins.gradle.org/m2/" }
-  }
-  dependencies {
-    classpath 'com.netflix.spinnaker.gradle:spinnaker-gradle-project:4.0.0'
-  }
+plugins {
+  id 'io.spinnaker.project' version "$spinnakerGradleVersion" apply false
 }
 
+apply plugin: "io.spinnaker.project"
+
 description = "Redis embedded server for Java integration testing"
 group = "com.netflix.spinnaker.embedded-redis"
-apply plugin: 'spinnaker.project'
-apply plugin: 'java'
-
-spinnaker {
-  dependenciesVersion = "0.160.0"
-}
+apply plugin: 'java-library'
 
 dependencies {
-  testCompile spinnaker.dependency("junit")
-  testCompile spinnaker.dependency('groovy')
+  testImplementation "junit:junit:4.12"
```

Comment:
For the record the dependencies come from https://github.com/spinnaker/spinnaker-dependencies/blob/v1.45.0/src/spinnaker-dependencies.yml (v1.45.0 is the latest tag there at the moment, which `com.netflix.spinnaker.gradle:spinnaker-gradle-project:4.0.0` brings in (or would bring in if we could resolve/use it) [here](https://github.com/spinnaker/spinnaker-gradle-project/blob/v4.0.0/src/main/groovy/com/netflix/spinnaker/gradle/dependency/SpinnakerDependency.groovy#L62).